### ISSUE TITLE
Privacy Considerations section - suggestions

### ISSUE THREAD

**Issue by GraniteKeep** [GlacierProtocol/glacierprotocol.github.io#19]
The protocol currently states:

> If privacy is very important to you, you might consider using a service like Shapeshift to exchange your Bitcoins for an more anonymous cryptocurrency, such as Monero, and then exchange them back to Bitcoins. However, this will cost you fees, and importantly, it requires you trust the operator of the exchange service not to steal or lose your funds.

Comments:
1. Shapeshift's privacy has recently been reduced (eliminated?) with KYC/AML requirements and similar services are likely to follow
2. Shapeshift has exchange limits far below the levels Glacier is aimed at

Also, just below that:

> This guide gives additional detail about how to increase Bitcoin anonymity using Monero & Tor.

Link is dead.

With the design document clearly stating that privacy is not a primary design goal of Glacier, are both suggestions better removed? Passing on the community's trust in Glacier to a third party like Shapeshift, or what will inevitably become an out-dated bitcoinnewsmagazine.com article, may not be wise.

I suggest we mention that bitcoin>monero>bitcoin is an option for privacy without giving any more detail, and that anything as specific as the link to the bitcoinnewsmagazine.com article is removed completely.

**Comment by bitcoinhodler** [comment]
I would go further and not even mention Monero by name,

**Comment by GraniteKeep** [comment]
Agreed. "Privacy coin" would provide enough of a pointer for individual research.
